### PR TITLE
AppDynamics and configuration parameter on S3

### PR DESCRIPTION
**Pull request by ehartung**
- In order to activate AppDynamics for exhibitor I added the needed configuration for STUPS tooling
- Made name of configuration in S3 bucket a separate parameter in order to be able to re-use a configuration when deploying a new ASG (no-downtime deployments like blue/green deployment)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +20,12 @@ Make sure that your got an unique ```APPLICATION_ID```
 
 ###### Deploy with Senza
 ```
-senza create exhibitor-appliance.yaml <STACK_VERSION> <APPLICATION_ID> <DOCKER_IMAGE_WITH_VERSION_TAG> <HOSTED_ZONE> $S3_BUCKET <MINT_BUCKET> <SCALYR_KEY> [--region AWS_REGION]
+senza create exhibitor-appliance.yaml <STACK_VERSION> <APPLICATION_ID> <DOCKER_IMAGE_WITH_VERSION_TAG> <HOSTED_ZONE> $S3_BUCKET <CONFIG_VERSION> <MINT_BUCKET> <SCALYR_KEY> <APPDYNAMICS_APP> [--region AWS_REGION]
```
Comment (hjacobs) [inline README.md]: you can now use keyword parameters in Senza (ApplicationId=...)

Comment (ehartung) [inline README.md]: OK, I will change this to use keyword parameters.

Comment (ehartung) [inline README.md]: Changed the documentation accordingly.

File: run.sh [hunk 2]
```diff
@@ -15,7 +15,8 @@ fi
 
 exec 2>&1
 
-java -jar /opt/exhibitor/exhibitor.jar \
+java -javaagent:/agents/appdynamics-jvm/javaagent.jar -Dappdynamics.agent.uniqueHostId=$(cat /agents/appdynamics-jvm/uniqueHostId) \
```
Comment (CyberDem0n) [inline run.sh]: It looks like you forgot to update Dockerfile. Can't find any mentions of appdynamics there.

Comment (ehartung) [inline run.sh]: There is no update in the Dockerfile needed, the AppDynamics java agent comes with Taupage.

Comment (valgog) [inline run.sh]: Until now there was no dependency on Tupage here, or do I understand it wrongly?

If so, I would prefer, that the script checks, if AppDynamics agent exists and if not, just work as before.

Comment (ehartung) [inline run.sh]: I made AppDynamics optional, i.e. if agent is not available or Senza parameter is not set.

**Comment by ehartung** [comment]
AppDynamics is not supported by Taupage anymore, therefore this pull-request is obsolete.

**Comment by hjacobs** [comment]
@ehartung I believe AppDynamics will be supported again, _hint hint_ :smirk:

**Pull request closed without merging**